xcpretty.gemspec

```diff
@@ -26,6 +26,8 @@ Gem::Specification.new do |spec|
   spec.test_files    = spec.files.grep(%r{^(test|spec|features)/})
   spec.require_paths = ["lib"]
 
+  spec.add_dependency 'rouge', '~> 1.8'
+
```

Comment:
Sorry for the confusion guys, multitasking :)
Let me try once again - if I fail let's just :ship: :it: .

Currently we use RubyGems, which is bad.
__We should be using Homebrew__ and treat `xcpretty` like a CLI tool rather than a library.

No dependencies is great because install time is fast, and rubygems are super slow with `dependencies->dependencies->dependencies`.

Not having that line in `.gemspec` gives you the possibility of just installing `xcpretty` and manually installing `rouge`.
If we put that line into `.gemspec`, you'll have Rouge installed on each Travis job.